app/views/foreman_cve_scanner/job_templates/run_cve_scanner.erb

```diff
@@ -53,7 +53,7 @@ scanners = {
 }
 
 if scanner == 'trivy'
-    cmd = "#{scanners[:trivy][target]} #{path}"
+    cmd = "#{scanners[:trivy][target]} --exit-code 0 #{path}"
```

Comment:
Nice. Thanks. Can you add this to the line below (options += "....")

Reply:
Applied.